action.yaml

```diff
@@ -52,6 +52,9 @@ inputs:
     description: Username used by StackStorm standalone authentication
   st2_auth_password:
     description: Password used by StackStorm standalone authentication
+  st2_packs:
+    description: String. Comma separated list of packs to install. This flag does not work with a --python3 only pack. (default `"st2"`). If you modify this option, be sure to also include `st2` in the list.
+    default: "st2"
```

Comment:
That'll work.
Curious though how hard could it be to specify a list of packs instead of comma-separated string? 🤔

In real use there are always more than a couple of packs in the system and one-liner is harder to manage. Just as an example, a real string of packs might look this way: 
```
st2_packs: "st2=v1.0.7,circle_ci=v0.5.3,packagecloud=v1.0.1,aws=v1.2.1,sensu=v0.5.2,slack=v0.12.7,github=v1.0.0,https://github.com/StackStorm/GitHubWebhook,https://github.com/StackStorm/st2ci,https://github.com/StackStorm/st2cd,windows=v0.6.0,https://github.com/StackStorm/stackstorm-st2community"
```
Having a code diff for a string like this might be confusing.

Reply:
good point. https://github.com/bitovi/github-actions-deploy-stackstorm/issues/20